File: grimorio de estudos/Cs50/Problem Sets/sentimental-mario-less/mario.py
from cs50 import get_int

# Always checks for h, if h is correct, break
while True:
    h = get_int("Please input the pyramid's desired height between 1 and 8: ")
    if h <= 8 and h >= 1:
        break

# Nested loop, for height then width
for x in range(h):
    for y in range(h):
        # Math mambo jumbo that does the logic for printing the pyramid
        if x + y >= h - 1:
            print("#", end="")
        else:
            print(" ", end="")
    print("")

File: harvardCs50/2022/Problem Sets/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h = 0;
    //ask for height input
    do
    {
        h = get_int("Please input the pyramid's desired height between 1 and 8: ");
        //printf("Height: %i\n", h);
    } while (h < 1 || h > 8 );

    //do pyramidwget

    //for loop for height of row
    for (int x = 0; x < h; x++)
    {
        //for loop to width
        for (int y = 0; y < h; y++)
        {
            //hashes
            if (x + y >= h - 1)
                printf("#");

            //add spaces
            else
            printf(" ");
        }
        printf("\n");
    }
}